### PR TITLE
Update team-config.yml for CUI Reasonable Adjustments

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: team-config.yml [hunk 1]
```diff
@@ -784,3 +784,12 @@ help-with-fees:
     build_notices_channel: "#pet-devops"
   tags:
     application: help-with-fees
+cui-ra:
```
Comment (thomast1906) [inline team-config.yml:787]: what is cui-ra? could it have a more meaningful name?

**Review by danlysiak — DISMISSED** [review]

File: team-config.yml (hunk 1 above)
Comment (kiran-yenigala-hmcts) [reply team-config.yml:787]: Citizen UI Reasonable Adjustments

**Review by danlysiak — APPROVED** [review]

File: team-config.yml (hunk 1 above)
Comment (thomast1906) [reply team-config.yml]: why not keep cui-ra?

Comment (kiran-yenigala-hmcts) [reply team-config.yml]: CUI is product name and ra is service name, just want to differentiate this

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**